.github/workflows/build.yml

```diff
@@ -173,7 +173,7 @@ jobs:
       matrix:
         goos: [ darwin ]
         goarch: [ "amd64", "arm64" ]
-        go: [ "1.17.5" ]
+        go: [ "1.18.1" ]
```

Comment:
nothing actionable now but pointing out that this may cause merge conflicts in ENT 📚

Reply:
Is it because ENT is on a different GO version? Is that intentional?

Reply:
`.circleci`, `.github`, `.release`, and `GNUMakefile` usually have collisions in general because of required divergences